### ISSUE TITLE
Foraged mushroom consumption

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9919]
**Preferred term label:**
Foraged mushroom consumption

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Monomethylhydrazine containing mushroom toxicity

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000304
- label: Recent foraged mushroom consumption
- parents: [Recent past medical history (HP:4000144)](https://hpo.jax.org/app/browse/term/HP:4000144)
- pmid: PMID:29262102
- def: Medical history of recently eating wild mushrooms.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9919
- synonyms:
